PackageExplorer/ViewModels/PortableLibraryViewModel.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Windows.Input;
+using NuGet.Frameworks;
+
+namespace PackageExplorer.ViewModels
+{
+	public class PortableLibraryViewModel
```

Comment:
This class is part of the MVVM pattern that allowed for separation of the code from the UI.  I'll add some comments to this effect.